projects/vdk-plugins/vdk-notebook/.plugin-ci.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+image: "python:3.7"
+
+.build-vdk-notebook:
+  variables:
+    PLUGIN_NAME: vdk-notebook
+  extends: .build-plugin
+
+build-py37-vdk-notebook:
+  extends: .build-vdk-notebook
+  image: "python:3.7"
+
+
+build-py38-vdk-notebook:
+  extends: .build-vdk-notebook
+  image: "python:3.8"
+
+build-py39-vdk-notebook:
+  extends: .build-vdk-notebook
+  image: "python:3.9"
+
+build-py310-vdk-notebook:
+  extends: .build-vdk-notebook
+  image: "python:3.10"
```

Comment:
Since we have an on going initiative for VDK support of Python 3.11, and we want to enable all vdk plugins to perform cicd testing using python:3.11, maybe directly include it with the newly added plugins? 
It's just added, so we can just handle it -in favour of tech debt.